github-action/src/main.ts

```diff
@@ -33,10 +33,18 @@ export async function runMain(): Promise<void> {
 			);
 			return;
 		}
-		const devContainerCliInstalled = await devcontainer.isCliInstalled(exec);
+		const specifiedDevContainerCliVersion =
+			core.getInput('cliVersion') ?? 'latest';
```

Comment:
Still use '0' as the default, so we can introduce breaking changes on the CLI by increasing its major version. Please use a constant, so we have it only once.

Reply:
I have introduced a constant to be referenced as the fallback version if no version is specified.